src/types/page.urls.ts

```diff
@@ -79,5 +79,7 @@ export const WRONG_RO =  ACTIVE_SUBMISSION_BASE + Templates.WRONG_RO;
 export const WRONG_RO_PATH = CONFIRMATION_STATEMENT + WRONG_RO;
 export const WRONG_REGISTER_LOCATIONS = ACTIVE_SUBMISSION_BASE + Templates.WRONG_REGISTER_LOCATIONS;
 export const WRONG_REGISTER_LOCATIONS_PATH = CONFIRMATION_STATEMENT + WRONG_REGISTER_LOCATIONS;
-export const WRONG_DETAILS = ACTIVE_SUBMISSION_BASE + Templates.WRONG_DETAILS;
+export const WRONG_DETAILS = ACTIVE_SUBMISSION_BASE + "wrong-officer-details";
```

Comment:
So would then suggest refactoring/renaming this to WRONG_OFFICER_DETAILS too.

Reply:
The template is used by both the officers and psc page so needs to stay as generic as possible.

Reply:
Ignore my reply changing this.